### PR TITLE
Fixes #23958: Abort server build if rudder.war is less than 1Mb

### PR DESCRIPTION
**Pull request by VinceMacBuche**
https://issues.rudder.io/issues/23958

### CODE REVIEW

**Comment by VinceMacBuche** [comment]
Commit modified

File: rudder-server/SOURCES/mvn-like [hunk 1]
```diff
@@ -28,3 +28,10 @@ else
 fi
 curl -s -o "${outfile}" "${url}"
 
+# Set a minimum size of 1Mb
+minimumsize=1000000
+actualsize=$(wc -c <"${outfile}")
+if [ ${actualsize} -le ${minimumsize} ]; then
```
Comment (Fdall) [inline rudder-server/SOURCES/mvn-like:34]: why not use `stat -c %s `, `wc` does not seem evident to me

Comment (VinceMacBuche) [reply rudder-server/SOURCES/mvn-like:34]: wc -c is just counting the bytes, it's in the usual answers about checking size of a file.

**Review by Fdall — APPROVED** [review]

File: rudder-server/SOURCES/mvn-like [hunk 2]
```diff
@@ -28,3 +28,10 @@ else
 fi
 curl -s -o "${outfile}" "${url}"
 
+# Set a minimum size of 1Mb
+minimumsize=1000000
```
Comment (clarktsiory) [inline rudder-server/SOURCES/mvn-like:32]: 1Mb is strictly 1024*1024 = 1048576, but I guess both values are close so it's fine ^^

**Pull request merged**